web_src/css/dashboard.css

```diff
@@ -88,7 +88,8 @@
   padding-right: 0.5rem;
 }
 
-.dashboard .dashboard-navbar .org-visibility .label {
+.dashboard .dashboard-navbar .org-visibility .label,
+.dashboard .repo-owner-name-list .org-visibility .label {
```

Comment:
`.dashboard .org-visibility .label`

?

And, can we just use `<div class="ui medium basic label org-visibility">` instead?

Reply:
And one more thing, since it is a Vue SFC, could its styles be scoped inside the Vue SFC?

Reply:
removed div, used `class="ui medium basic label org-visibility"`, and put style in scoped style in https://github.com/go-gitea/gitea/pull/24558/commits/12d45ddcafad0d794d0c65dcba4367247eaaab79

Reply:
I guess I can do some clean up. Will try